content/contributing/helping-out/writing-docs.md

```diff
@@ -0,0 +1,237 @@
+
+This is intended to cater to two different types of learners, without compromising the experience for either:
+
+* **Example-first:** These users want to dive right in, see everything in action and get a working game as quickly as possible.
+These users often have an idea in their mind that they want to start prototyping as quickly as possible.
+* **Definition-first:** These users want to carefully build up a mental model of Bevy, thoroughly understanding each new concept before moving on.
+These users tend to be driven by curiosity, or are aiming to carefully develop a new skill.
+
+Crucially, these paths are independent of the experience levels of the learner!
+Bevy intentionally aims to be inclusive of both complete beginners who have never programmed before, and professionals coming from other engines.
+
+| | **Beginner** | **Professional** |
+|-|-|-|
+| **Example-first** | Enthusiastic, wants to create a new version of the game they love. | Exploratory, wants to dive in and see how Bevy holds up in practice. |
+| **Definition-first** | Curious, wants to understand how making games works. | Critical, wants to understand Bevy's unique design choices. |
+
+Each of these requires their own complementary learning paths that branch as soon as they get to the [Learn page](@/learn/_index.md) to ensure that the first experience that they have with Bevy matches what they need.
+
+Once users have completed the introductory learning materials in their path of choice, they can begin creating their own games or move on to our advanced examples to see how everything comes together in a realistic way.
+
+### Bevy Quick Start: the example-first path
+
+Users following the example-first path will tend to take the following route:
+
+1. Encounter the Bevy homepage due to social media or word of mouth.
+2. Navigate to the Learn page.
+3. Choose one of the most relevant **quick start games**.
+4. Complete that tutorial.
+5. Dive into making the game they have in mind, accessing the following resources as needed when they encounter road-blocks:
+   1. Official Examples.
+   2. The Bevy book.
+   3. Community tutorials and template games.
+   4. Various community support forums.
+   5. Streams, YouTube channels and blogs.
+   6. Advanced examples.
+
+Each quick start game should:
+
+1. Assume zero existing knowledge of Bevy.
+2. Begin with a initial high-level explanation of what we're trying to build.
+3. Introduce commented code first, then explain what each of the critical terms means as they come up.
+4. Be broken into compilable, playable sections: one per page of the guide.
+5. Gradually refactor the code to add more functionality.
+6. End with a list of suggestions (with appropriate links) on how you could extend the game in interesting ways.
+
+This path should prioritize:
+
+1. Rapid time-to-fun.
+2. Functional, good-enough explanations that are tied to the code in front of them.
+3. Relevance of quick-start game to the genre of game they want to make.
```

Comment:
```suggestion
3. Relevance of quick start game to the genre of game they want to make.
```
This should either be only two separate words, or only hyphenated, for consistency; not a mixture of both.